lib/lodqa/bs_client.rb

```diff
@@ -55,8 +56,72 @@ def subscribe ws, request_id, url, mode
         end
       end
 
+      def sparqls pgp, mappings, graph_finder, enum
+        anchored_pgps(pgp, mappings).each do |anchored_pgp|
+          to_sparql(anchored_pgp, graph_finder){ |sparql| enum << sparql}
+        end
```

Comment:
ここも内部的にEnumeratorを使っています。
配列をつくって返すようにすれば引数enumが不要になります。

```suggestion
        sparqls = []
 
        anchored_pgps(pgp, mappings).each do |anchored_pgp|
          to_sparql(anchored_pgp, graph_finder){ |sparql| sparqls << sparql}
        end
        
        sparqls
```

Reply:
対応しました。